docs/essentials/secure-storage.md

```diff
@@ -12,6 +12,28 @@ ms.date: 05/04/2018
 
 The **SecureStorage** class helps securely store simple key/value pairs.
 
+## Getting Started
+
+To access the **SecureStorage** functionality the following platform specific setup is required.
+
+# [Android](#tab/android)
+
+No additional setup required.
+
+# [iOS](#tab/ios)
+
+When developing on the iOS Simulator it is required to enable the Keychain entitlement and add a Keychain group with the application's package identifier.
+
+Open the Entitlements.plist in the iOS project and find the Keychain entitlement and enable it. This will automatically add the application's identifier as a group.
+
+In the project properties under iOS Bundle Signing set the **Entitlements.plist** in the Custom Entitlements.
```

Comment:
- "properties, under **iOS Bundle Signing**"
- "set **Custom Entitlements** to **Entitlements.plist**."